### PR TITLE
hw3 completed

### PR DESCRIPTION
**Pull request by strrust**

### CODE REVIEW

File: src/test/java/ru/training/at/hw3/DefaultTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package ru.training.at.hw3;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+import java.util.concurrent.TimeUnit;
+
+public abstract class DefaultTest {
```
Comment (komyagin) [inline src/test/java/ru/training/at/hw3/DefaultTest.java:10]: It's not good name for common class

File: src/test/java/ru/training/at/hw3/GetDataFromProperties.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package ru.training.at.hw3;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class GetDataFromProperties {
```
Comment (komyagin) [inline src/test/java/ru/training/at/hw3/GetDataFromProperties.java:7]: Don't use verbs in the class name. PropertiesLoader maybe?

File: src/test/java/ru/training/at/hw3/ex1/HomePageTest.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        mainPageIframes = new MainPageIframes(webDriver);
+    }
+
+    @Test
+    public void test() {
+        mainPage.openPage();
+        mainPage.checkTitle();
+
+        loginWindow.openLoginWindow();
+        loginWindow.inputName(GetDataFromProperties.getProperty("loginName"));
```
Comment (komyagin) [inline src/test/java/ru/training/at/hw3/ex1/HomePageTest.java:36]: You can import static method getProperty()

**Review by komyagin — CHANGES_REQUESTED** [review]

File: src/test/java/ru/training/at/hw3/DefaultTest.java (hunk 1 above)
Comment (strrust) [reply src/test/java/ru/training/at/hw3/DefaultTest.java:10]: Renamed to TestBase

File: src/test/java/ru/training/at/hw3/GetDataFromProperties.java (hunk 2 above)
Comment (strrust) [reply src/test/java/ru/training/at/hw3/GetDataFromProperties.java:7]: good suggestion! renamed to PropertiesLoader

File: src/test/java/ru/training/at/hw3/ex1/HomePageTest.java (hunk 3 above)
Comment (strrust) [reply src/test/java/ru/training/at/hw3/ex1/HomePageTest.java:36]: Changed

**Review by komyagin — APPROVED** [review]

**Pull request merged**